ThunderCloud/StormLanguageController.swift

```diff
@@ -208,7 +208,6 @@ public class StormLanguageController: NSObject {
                     //Set the major language if only the language matches. Major language pack always exists if a minor one exists
                     if let languageCode = pack.locale.languageCode, let languageName = pack.fileName.components(separatedBy: "_").first {
                         majorLanguagePack = LanguagePack(locale: Locale(identifier: languageCode), fileName: languageName)
-                        return (regionalLanguagePack: nil, majorLanguagePack: majorLanguagePack)
```

Comment:
Does removing this not mean we could get landed with a major language pack of, say `en` but a minot language pack of say: `es` so `es_en` with overrides taken from the Spanish regional pack?

Reply:
Yeah I think you might be right @simonmitchell

Reply:
So this line here is actually one I added in on Monday which was causing an issue with the region detection. If you check back to be1460677266c64f5c2365c3174f8cfff1f90335 you'll see where I've added it :)